internal/signer/darwin/keychain/keychain.go

```diff
@@ -405,3 +439,91 @@ func certIn(xc *x509.Certificate, xcs []*x509.Certificate) bool {
 	}
 	return false
 }
+
+func (k *Key) EncryptRSA(hashInput hash.Hash, random io.Reader, msg []byte) ([]byte, error) {
```

Comment:
Make this a private method.

Let's remove the hash and random params from the function signature.

Reply:
Could we actually delete this function since the Encrypt() can also encrypt RSA? I created this function when I was picking whether to use the SecKey API or Go's crypto API so this is outdated. (I'll also make a note to next time remove outdated things before I PR)

Reply:
I think it's a nice touch to split it into a separate method, and then the public method can just call Encrypt RSA if it's a RSA key.